docs/esp32/quickref.rst

```diff
@@ -340,6 +340,13 @@ Notes:
 * Calling ``deepsleep()`` without an argument will put the device to sleep
   indefinitely
 * A software reset does not change the reset cause
+* There may be some leakage current flowing through enabled internal pullups.
+  To further reduce power consumption it is possible to disable the internal pullups::
+
+    p1 = Pin(4, Pin.IN, Pin.PULL_HOLD)
+    # or
+    p2 = Pin(4, Pin.IN)
+    esp32.gpio_hold(p2, True)
```

Comment:
gpio_hold() was not merged, just the PULL_HOLD functionality, so please remove this second part.

Also, upon leaving deepsleep (ie after a reset) it may be necessary to un-hold the pin explicitly via `Pin(4, Pin.OUT, None)`